e2e/e2e_test.go

```diff
@@ -320,6 +322,44 @@ func testPionE2EMTUs(t *testing.T, server, client func(*comm)) {
 	}
 }
 
+func testPionE2ESimpleED25519(t *testing.T, server, client func(*comm)) {
+	lim := test.TimeOut(time.Second * 30)
+	defer lim.Stop()
+
+	report := test.CheckRoutines(t)
+	defer report()
+
+	for _, cipherSuite := range []dtls.CipherSuiteID{
+		dtls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+		dtls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+		dtls.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
```

Comment:
I've not (re)included the `CCM` ciphers here which we had in the original because according to `openssl ciphers` it's not actually there. Even though the documentation suggests it should be. Not sure what's going on there.

I'll try and enable them anyway after we've figured out why the tests are failing on the definitely supported suites and see what happens.

Reply:
They seem to work just fine in the client tests, so added them back.